### DISCUSSION TITLE
How to change base Accordion's background color?

### DISCUSSION THREAD

**Discussion by Reymdusk** [reactstrap/reactstrap#2586] [Q&A] [answered]
Hello,

I cannot seem to find a way to change the background color of the Accordion component. 
There is nothing about this in the Storybook.

I tried to add `style={{backgroundColor: "black"}}` to the `<Accordion>` tag. 
I even tried the `flush` option to see if removing the default background, as the Storybook suggests, to try to edit the color. 
I also tried to change the contents of the items inside the `AccordionHeader` tag, but it only changes the item itself, even if I remove margin / padding. 

I'm trying to use a dark-theme, and the Accordion's very bright default color is not what I want.

Note that I can change the styling for the `AccordionBody` tag, and it works well, but I want the Header, and/or the overall Accordion to be styled. 

Can someone assist?

**Comment by Reymdusk** [comment, marked as answer]
Figured it out, had to edit the `.accordion-button` class instead

**Reply by kschmott** [reply]
Are you using nextjs? I'm having the same problem and I can't seem to edit that class

**Reply by Reymdusk** [reply]
Unfortunately, no.
Have you attempted with CSS Modules?

**Reply by kschmott** [reply]
Yeah, I can't seem to change the styles when it is focused or :not(.collapsed)

**Reply by kschmott** [reply]
Managed to fix it by copying the css from the browser. My css file looks like this: 
![image](https://user-images.githubusercontent.com/103714838/185477109-2d220f22-41b9-49a8-9775-4409133ccea0.png)
